bbn-test-4/covenant-committee/README.md

```diff
@@ -0,0 +1,11 @@
+# Covenant Committee
```

Comment:
Should we add a reference to this file on `bbn-test-4/README.md`?

Reply:
Done!